src/test/resources/pacts/publicapi-direct-debit-connector-search-mandates-endpoint.json

```diff
@@ -0,0 +1,216 @@
+{
+  "consumer": {
+    "name": "publicapi"
+  },
+  "provider": {
+    "name": "connector"
+  },
+  "interactions": [
+    {
+      "description": "search mandates by parameters",
+      "providerStates": [
+        {
+          "name": "a gateway account with an external id and a confirmed mandate searchable by all parameters",
+          "params": {
+            "gateway_account_id": "2po9ycynwq8yxdgg2qwq9e9qpyrtre",
+            "mandate_id": "jkdjsvd8f78ffkwfek2a",
+            "unique_identifier": "AAAAA",
+            "charge_id": "charge97837509646393e3A",
+            "reference": "a reference",
+            "state":"pending",
+            "bank_statement_reference": "a bstatement ref",
+            "name":"a name",
+            "email": "rory@example.com",
+            "from_date":"2018-07-11T12:30:48.317Z",
+            "to_date": "2020-07-11T12:30:48.317Z",
+            "page":"1",
+            "display_size": "1"
+          }
+        }
+      ],
+      "request": {
+        "method": "GET",
+        "path": "/v1/api/accounts/123456/mandates",
+        "query": "email=rory@example.com&name=a%20name&state=pending&bank_statement_reference=a%20bstatement%20ref&reference=a%20reference&from_date=2018-07-11T12:30:48.317Z&to_date=2020-07-11T12:30:48.317Z&page=1&display_size=1",
```

Comment:
I think its clearer to specify the query using the hash notation rather than a single string. Take a look at the docs, if it doesn't work then lets keep it as-is with a String.
https://docs.pact.io/getting_started/matching#query-params